### DISCUSSION TITLE
Date Ranges

### DISCUSSION THREAD

**Discussion by tateforgey** [vzhd1701/csv2notion#14] [Q&A] [answered]
Thank you for this incredible effort. It is allowing me to use Notion in ways I only dreamed of.

Is it possible to import date ranges? Notion's date fields have the capability of having a start and end date in the same field, but I haven't been able to figure out a syntax where csv2notion recognizes that as valid. A small detail, but it would be nice. Thanks.

**Comment by vzhd1701** [comment, marked as answer]
Done, check out the new version.
You can set ranges using comma separator.

```
name,date
test1,"2022-12-01,2022-12-05"
```

**Reply by tateforgey** [reply]
You are a rockstar!

At first I had put literal quotes around the range in the Google Sheet I was working with and that threw a string error after exporting to CSV, but when I removed those and re-exported it worked great. Sweet!

I've seen a bunch of comments around the web about the time always being included in Notion imports. So, I assume there is no way around that. It is a bit of an annoyance, but now that I have the correct ranges these records will all display correctly in calendar views. So, I am a happy man.

This is a game changer. Your tool helps Notion behave like the relational database it is and was the missing piece in helping me consolidate all the disparate data in my life into one system. Thanks!
